tests/pkg_ubasic/tests/01-run.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    for i in range(1,6):
```

Comment:
The static checks are whining about a missing space.
```suggestion
    for i in range(1, 6):
```

Reply:
Fixed and squashed